schemas/questions/types/calculated.json

```diff
@@ -59,7 +89,7 @@
             },
             "answers_to_calculate": {
               "type": "array",
-              "minItems": 2,
+              "minItems": 1,
```

Comment:
Based on the changes to `calculated_question_validator.py`, will this now allow a list of non-dynamic sources with a length of 1?

Reply:
It will fail validation in python if it's static answers ids list with len of 1 now. So the schema won't get through and fail.